pom.xml

```diff
@@ -400,6 +414,20 @@
                   </manifestEntries>
                 </transformer>
               </transformers>
+              <filters>
+                <filter>
+                  <artifact>org.folio:raml-module-builder</artifact>
```

Comment:
Please, update this parameter according to the comment https://github.com/folio-org/mod-finance/pull/4#discussion_r323089371